bot/utils/recognition.py

```diff
@@ -100,20 +102,50 @@ def _check_google_speech_response(response):
     return None
 
 
-def get_text_from_speech(file_id):
-    """Gets text from voice."""
+def _get_converted_video_content(buffer: bytes) -> bytes:
+    with NamedTemporaryFile(suffix=MP4) as tmp_video:
+        tmp_video.write(buffer)
+
+        tmp_video_filename_mp4 = tmp_video.name
+        tmp_audio_filename_wav = tmp_video_filename_mp4[: -len(MP4)] + ".wav"
+
+        subprocess.call(
```

Comment:
Ich bin fast sicher, dass App Container beinhaltet keine `ffmpeg`, oder? @AlexanderSutul hast du die Container bauen versucht?

In jedem Fall muss dieser Anruf drin `try \ catch` sein.

Reply:
@egregors Ich denke, wir installieren `ffmpeg` in unserem `docker image`.  Und Ich werde hinzufügen `try \ catch`. Kein Probleme.

Reply:
@egregors Ich habe alles gemacht. Darfst du bitte sehen?